03_oop_part_2/src/page.ts

```diff
@@ -0,0 +1,15 @@
+export class Page {
+    private pageNumber: number;
+    private pageType: string;
+    private pageMaterial: string;
```

Comment:
@romasaldan, it is better to use an enum, because the property has a defined set of values ("simple paper", "glossy paper")